package.json

```diff
@@ -33,7 +33,9 @@
     "zone.js": "^0.8.4"
   },
   "devDependencies": {
-    "@ionic/app-scripts": "1.2.4",
+    "@ionic/app-scripts": "1.3.2",
+    "@ionic/cli-plugin-cordova": "0.0.12",
```

Comment:
what are these dependencies? what do they do/replace/intended for? I can't find anything here https://www.npmjs.com/package/@ionic/cli-plugin-cordova or in the cli repo https://github.com/driftyco/ionic-cli